Packs/ZeroTrustAnalyticsPlatform/Integrations/ZeroTrustAnalyticsPlatform/ZeroTrustAnalyticsPlatform.py

```diff
@@ -365,85 +360,34 @@ def comments_to_notes(client: Client, comments: List):
     Turns comments into XSOAR entries
     """
 
-    def to_note(comment, filenames):
+    def to_note(comment):
         occurred = comment["datetime_created"]
         oid = comment["id"]
-        readable = {
-            "occurred": occurred,
-            "contents": comment["comment"],
-            "type": "comment",
-            "files": filenames,
-        }
+        user_display = user_to_display(comment["user"])
+        footer = f"\n\nSent by {user_display} via ZTAP"
         return {
             "Type": EntryType.NOTE,
             "ContentsFormat": EntryFormat.JSON,
             "Contents": comment,
-            "HumanReadable": readable,
-            "ReadableContentsFormat": EntryFormat.JSON,
+            "HumanReadable": comment["comment"] + footer,
```

Comment:
```suggestion
            "HumanReadable": comment.get("comment") + footer,
```

You can also use f-strings here